src/main/java/com/google/sps/Marker.java

```diff
@@ -0,0 +1,154 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Hashtable;
+import java.util.Set;
+
+/**
+ * Marker is the container class for when an unique marker is createsof people are meeting and are therefore
+ * busy. The marker's title, description, location, categories are considered read-only. All the other attributes 
+ * should be modified by the actions of the viewer users.
+ */
+
+public class Marker {
+  private final String title;
+  private final String description;
+  private final Hashtable<String, Double> location = Hashtable<String, Double>(2);
+  private final Set<String> categories = new HashSet<String>();
+  private Set<String> flags = new HashSet<String>();
```

Comment:
ArrayList not HashSet, and explain that the string content is the comment/reason for flagging

Reply:
Thank you